demos/demo_operation_log.sh

```diff
@@ -51,3 +46,5 @@ run_command "jj op restore $rebase_op"
 comment "We're now back to before the \"other stuff\"
 change existed:"
```

Comment:
More optional, perhaps we should mention that `undo` didn't undo the checkout operation. So, instead of "We can also see what the repo looked like...", it could say something like:

> Note that only the rebase was undone, and the subsequent "other stuff" change was not undone.

> Let's say we instead want to go back to the state of the repo right *after* the rebase.

(Now run `jj op restore` as in the script).

Reply:
I think the reason I added `--at-op` was not because it's commonly used, but more as a way of showing the user that the past states of the repo is available. I don't know if it's worth the extra space it takes. Makes sense to explain that the checkout is not undone, and to say that undo is an operation in itself.